src/zenml/cli/base.py

```diff
@@ -63,19 +66,116 @@ def init(path: Optional[Path]) -> None:
     )
 
 
-@cli.command("clean")
-@click.option("--yes", "-y", type=click.BOOL, default=False)
-def clean(yes: bool = False) -> None:
-    """Clean everything in repository.
+def _delete_local_artifact_metadata(force_delete: bool = False) -> None:
+    """Delete local metadata and artifact stores from the active stack.
 
     Args:
-      yes: bool:  (Default value = False)
+      force_delete: Whether to force delete the metadata and artifact stores."""
+    if not force_delete:
+        confirm = confirmation(
+            "DANGER: This will completely delete anything inside the folders for the following stack components: \n"
+            "- local metadata store \n"
+            "- local artifact store. \n\n"
+            "Are you sure you want to proceed?"
+        )
+        if not confirm:
+            declare("Aborting clean.")
+            return
+
+    repo = Repository()
+    if repo.active_stack:
+        metadata_store_path = (
+            Path(repo.active_stack.metadata_store.local_path)
+            if repo.active_stack.metadata_store.local_path
+            else None
+        )
+        artifact_store_path = Path(repo.active_stack.artifact_store.path)
+        if (
+            metadata_store_path
+            and not is_remote(str(metadata_store_path))
+            and not is_remote(str(artifact_store_path))
+        ):
+            # delete all files inside those directories
+            for path in metadata_store_path.iterdir():
```

Comment:
Can't we just delete the entire metadata dir here instead of looping over it?

Reply:
@stefannica said that he just wanted to delete the files inside the directories but not the directories themselves, if I didn't mishear him.

Reply:
Can you explain why @stefannica ?